src/funding/funding.js

```diff
@@ -27,30 +27,30 @@ export function isFundingEligible(source : $Values<typeof FUNDING>, { layout, pl
         return false;
     }
 
-    if (layout && fundingConfig.layouts && fundingConfig.layouts.indexOf(layout) === -1) {
+    if (layout && fundingConfig.layouts && fundingConfig.layouts.indexOf(layout) === -1 && !enableFunding.some(funding => funding === fundingSource)) {
```

Comment:
This should not be necessary -- the APMs will continue to be eligible in standalone and vertical modes right?

Reply:
Still don't think this is required. This logic will not apply to standalone.